### PR TITLE
Enhance PAN-OS commits

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/41980)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml [hunk 1]
```diff
@@ -173,6 +173,31 @@ script:
   - arguments:
     - description: Commit description.
       name: description
+    - description: To commit admin-level changes on a firewall, include the administrator
+        name in the request.
+      name: admin_name
+    - auto: PREDEFINED
+      description: (True/False) Force Commit
```
Comment (Shellyber) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.yml:180]:
```suggestion
      description: Force Commit.
```
No need for the True/False in the description.

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml [hunk 2]
```diff
@@ -173,6 +173,31 @@ script:
...
+        - 'false'
+    - auto: PREDEFINED
+      description: (True/False) Partial commit while excluding shared objects and device and network configuration.
+      name: exclude_device_network_configuration
+      defaultValue: 'false'
+      predefined:
+        - 'true'
+        - 'false'
+    - auto: PREDEFINED
+      description: (True/False) Partial commit while excluding shared objects and
```
Comment (Shellyber) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.yml:194]: The description here should be the same as `exclude_device_network_configuration`?

**Review by Shellyber — COMMENTED** [review]
Looking good 👍

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 3]
```diff
@@ -602,7 +632,13 @@ def panorama_commit_status_command(args: dict):
 @logger
 def panorama_push_to_device_group(args: dict):
     command: str = ''
-    command += f'<device-group><entry name="{DEVICE_GROUP}"/></device-group>'
+    device_group = args.get('device-group', DEVICE_GROUP)
```
Comment (Shellyber) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:635]: Why this argument is needed? the device group is taken from the configuration, no? 
Also, you didn't add it in the yml file.

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 4]
```diff
@@ -602,7 +632,13 @@ def panorama_commit_status_command(args: dict):
...
+    command += f'<device-group><entry name="{device_group}"/></device-group>'
+
+    serial_number = args.get('serial_number')
+    if serial_number:
+        command = f'<device-group><entry name="{device_group}"><devices><entry name="{serial_number}"/>' \
```
Comment (Shellyber) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:640]: shouldn't be += ?

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 3 above)
Comment (DinaMeylakh) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:635]: It's already there, but you're right this is redundant

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml (hunk 2 above)
Comment (DinaMeylakh) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.yml]: good catch, thanks

File: Packs/PAN-OS/Integrations/Panorama/README.md [hunk 5]
```diff
@@ -268,7 +268,11 @@ Commits a configuration to Palo Alto Firewall or Panorama, but does not validate
 
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| description | Commit description. | Optional | 
+| description | Commit description. | Optional |
+| admin_name | To commit admin-level changes on a firewall, include the administrator name in the request. | Optional |
+| force_commit | (True/False) Force Commit | Optional |
+| exclude_device_network_configuration | (True/False) Partial commit while excluding shared objects and device and network configuration. | Optional | 
+| exclude_shared_objects | (True/False) Partial commit while excluding shared objects and device and network configuration.| Optional |
```
Comment (Shellyber) [inline Packs/PAN-OS/Integrations/Panorama/README.md:275]: Let's update the readme as well with all the changes

**Review by Shellyber — APPROVED** [review]

**Comment by Itay4** [comment]
build fails on non related and inconsistent test
merging

**Pull request merged**